frontend/src/App.tsx

```diff
@@ -916,9 +925,37 @@ export class App extends PureComponent<Props, State> {
       queryString = queryString.substring(1)
     }
 
+    const baseUriParts =
+      this.connectionManager && this.connectionManager.getBaseUriParts()
+
+    if (!baseUriParts) {
```

Comment:
This is not super clear why the disconnection of the server is related to `baseUriParts` being falsy. What does `this.connectionManager.getBaseUriParts()` return if there is no base URI?

Reply:
`getBaseUriParts()` may return `undefined` if the `connectionManager` has been created but we no longer have an active websocket connection to the server.

I'll add an explanatory comment.